api/api.go

```diff
@@ -532,6 +532,12 @@ func (b *BlockChainAPI) Call(
 		return nil, err
 	}
 
+	// Default to "latest" block tag
+	if blockNumberOrHash == nil {
+		latest := rpc.BlockNumberOrHashWithNumber(rpc.LatestBlockNumber)
```

Comment:
let's make this a global variable in API package

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/401/commits/7bb23ab3fc94718ecf220bfa2efc04afbdcd7e33